tests/rustdoc-ui/doctest/main-alongside-stmts.stderr

```diff
@@ -7,8 +7,8 @@ LL | //! ```
 warning: the `main` function of this doctest won't be run as it contains expressions at the top level, meaning that the whole doctest code will be wrapped in a function
   --> $DIR/main-alongside-stmts.rs:26:1
    |
-LL | //! ```
-   | ^^^^^^^
+LL | //!
+   | ^^^
```

Comment:
(unrelated, preexisting) Lol, we really gotta improve the span finding heuristic for all of these cases...

Reply:
Yeah, the spans are really rough. I think no one realized how bad they were until now.